libafl/src/events/llmp.rs

```diff
@@ -90,11 +90,11 @@ where
     }
 }
 
-impl<I, S, SP, ST> LlmpEventManager<I, S, SP, ST>
+impl<I, OT, S, SP, ST> LlmpEventManager<I, OT, S, SP, ST>
 where
     I: Input,
-    S: IfInteresting<I>,
-    SP: ShMemProvider,
+    OT: ObserversTuple,
+    SP: ShMemProvider + 'static,
```

Comment:
Does the 'static lifetime work for all providers? I guess yes

Reply:
that's copy pasted, idk